views/templates/admin/installmentSettings.tpl

```diff
@@ -105,6 +105,30 @@
             </div>
         </div>
 
+        <div class="paypal-form-group pp__flex-align-center pp_mb-20">
+            <div class="label">
+                {if isset($PayPal_sandbox_mode) && $PayPal_sandbox_mode}
+                    {l s='REST Secret ID Sandbox' mod='paypal'}
+
+                {else}
+                    {l s='REST Secret ID' mod='paypal'}
+                {/if}
+            </div>
+
+            <div class="configuration">
+                <div class="bootstrap pp__flex" style="width: 50%">
+                    <input
+                            type="text"
```

Comment:
It should be a type password, shouldn't it ?

Reply:
Updated on type password, i'm agree